example/fx.py

```diff
@@ -59,24 +62,26 @@ def __init__(self, leds):
                 lambda acc, pt: (max(acc[0], pt[0]), max(acc[1], pt[1])),
                 leds.values(), (float('-inf'), float('-inf')))
             self.env = FxEnv(Resolution(max_by_x, max_by_y))
-            self.colors = {x: (0, 0, 0) for x in leds}
+            self.colors = dict.fromkeys(leds, (0, 0, 0))
             self.empty = False
         else:
             self.empty = True
 
     def update(self, frame_time, fx):
         if self.empty:
             return
+
         self.env.time = frame_time
         for key in self.colors:
             self.colors[key] = ColorRgb.from_vec3(
-                *fx(self.env, self.leds[key][0], self.leds[key][1])).rgb
+                *fx(self.env,
+                    *self.leds[key][:2])).rgb
```

Comment:
I cannot agree that this is an improvement. I would like to see in this place an explicit passing of the parameters x and y, no need to make code shorter in this case.

Reply:
Fixed.